src/generator.ts

```diff
@@ -24,7 +24,7 @@ export interface GenerateOptions {
 	 * Control which PropTypes are included in the final result
 	 * @param proptype The current PropType about to be converted to text
 	 */
-	shouldInclude?(proptype: t.PropTypeNode): boolean | undefined;
+	shouldInclude?(proptype: t.PropTypeNode, component: t.ComponentNode): boolean | undefined;
```

Comment:
```suggestion
	shouldInclude?(proptype: t.PropTypeNode, component?: t.ComponentNode): boolean | undefined;
```

Could you also document this param please

Reply:
Why would it be `undefined` here?

Reply:
The other params aren't document either. I don't think there's anything meaningful to add. It will probably boil down to "the component of the prop" which is just repeating the code.

Reply:
nvm wrong place, done